### PR TITLE
spelling is hard

### PR DESCRIPTION
**Pull request by Bjvanminnen**

### CODE REVIEW

File: shared/test/test_assets.rb [hunk 1]
```diff
@@ -186,10 +186,10 @@ def test_viewing_abusive_assets
     FilesApi.any_instance.unstub(:admin?)
 
     # teacher can view
-    FilesApi.any_instance.stubs(:teaces_student?).returns(true)
+    FilesApi.any_instance.stubs(:teaches_student?).returns(true)
```
Comment (laurelfan) [inline shared/test/test_assets.rb:189]: I would be a bit worried about whether this test is actually testing what it says it is testing if it works both ways.. do you understand why that is?

Comment (Bjvanminnen) [inline shared/test/test_assets.rb:189]: Good point. I'll spend some more time looking at this.

File: shared/test/test_assets.rb [hunk 2]
```diff
@@ -186,10 +186,10 @@ def test_viewing_abusive_assets
...
 
     # teacher can view
-    FilesApi.any_instance.stubs(:teaces_student?).returns(true)
-    get(@assets, channel_id, asset_name)
-    assert @assets.last_response.successful?
-    FilesApi.any_instance.unstub(:teaces_student?)
+    FilesApi.any_instance.stubs(:teaches_student?).returns(true)
+    get(non_owner_assets, channel_id, asset_name)
+    assert non_owner_assets.last_response.successful?
+    FilesApi.any_instance.unstub(:teaches_student?)
```
Comment (Bjvanminnen) [inline shared/test/test_assets.rb:192]: The reason this was working anyways, is we were looking at assets for which we were the owner (so we don't even hit teaches_student). By changing to non-owner assets, the test fails when we stub teaces_student, and then passes again when correcting the typo.

**Comment by laurelfan** [comment]
cool, LGTM

**Comment by Bjvanminnen** [comment]
Thanks for catching this :)

**Pull request merged**